### PR TITLE
Fix bug where the dropdowns are only scrollable inside the toolbar

### PR DESCRIPTION
**Pull request by AGS-**
githubgraphiqlscrollbug

### CODE REVIEW

File: css/app.css [hunk 1]
```diff
@@ -61,7 +61,7 @@
 }
 
 .graphiql-container .toolbar {
-  overflow-x: auto;
+  overflow-x: visible;
```
Comment (wincent) [inline css/app.css]: Seems reasonable, but isn't `overflow: visible` the default? (Not sure if it's cascading down here though, haven't checked).

Comment (AGS-) [inline css/app.css]: It's defaulting to scroll for some reason, so I think hard coding it would be the best option.

Comment (wincent) [inline css/app.css]: Sounds fair.

**Comment by AGS-** [comment]
Thanks @wincent !

**Pull request merged**